Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2148,14 +2143,12 @@ void LoadDefaultBufferConfigurationSettings(
     unsigned int totalThreadCount;
     unsigned int threadUnit;
 
-#if defined(_WIN32) || defined(__linux__)
```

Comment:
Since in InitThreadManagmentParams, macOS is not considered. numGroups is 0 on macOS. So if `-ss` is not -1, will crash on line `coreCount /= numGroups;`
Please keep the macro here.
Let's don't support `-ss and -lp` on macOS for now.

Reply:
Okay